packages/regl-worldview/src/utils/getHitmapDefaults.js

```diff
@@ -0,0 +1,89 @@
+// @flow
+
+//  Copyright (c) 2018-present, GM Cruise LLC
+//
+//  This source code is licensed under the Apache License, Version 2.0,
+//  found in the LICENSE file in the root directory of this source tree.
+//  You may not use this file except in compliance with the License.
+
+import type { AssignNextIdsFn, MouseEventObject } from "../types";
+import { intToRGB } from "./commandUtils";
+
+export const nonInstancedGetHitmap = <T: Object>(
+  props: Array<T> | T,
+  assignNextIds: AssignNextIdsFn,
+  alreadySeenObjects: Array<MouseEventObject>
+) => {
+  const propsArray = Array.isArray(props) ? props : [props];
+  const hitmapArray: Array<T> = propsArray
+    .map((prop) => {
+      if (alreadySeenObjects.some(({ object }) => object === prop)) {
+        return null;
+      }
+      const hitmapProp = { ...prop };
+      const [id] = assignNextIds({ type: "single", callbackObject: prop });
+      const hitmapColor = intToRGB(id);
+      hitmapProp.color = hitmapColor;
+      if (hitmapProp.points) {
```

Comment:
should we check points.length > 0? I don't remember what the commands do in the case of `point: foo,  points: []`

Reply:
I'm pretty sure they just create a single point at the default position, but I'll add this line